frontend/user/vue-components/ApplicationView.vue

```diff
@@ -135,9 +135,9 @@
         return utils.maxIframeSize();
       },
       focusIframe: function() {
-        let iframe = this.$el.querySelector('iframe');
-        if(iframe !== null) {
-          iframe.focus();
+        if(this.$el.querySelector !== undefined && // In case $el is not rendered
```

Comment:
I think you are not using the right approach. if the element is not rendered, you should check if the element is not rendered. You are probing the existence of a routine that happens to be correlated to the existence of the element, but it's not the thing that is the core issue.